src/css/curCSS.js

```diff
@@ -35,7 +35,7 @@ function curCSS( elem, name, computed ) {
 		// but width seems to be reliably pixels.
 		// This is against the CSSOM draft spec:
 		// https://drafts.csswg.org/cssom/#resolved-values
-		if ( !support.pixelMarginRight() && rnumnonpx.test( ret ) && rmargin.test( name ) ) {
```

Comment:
So is it okay that `rboxStyle` now catches properties like `borderLeftWidth`? Our CSS unit tests don't seem to do much with the `border*Width` properties so I'm just wondering what effect we'll have in going through this path.

Reply:
It's fine because the swap logic only kicks in when gCS returns a non-px unitful value. And even if that does happen, the only consequence is taking a slower path (requiring layout calculation) to get pixels.